governance/polls/Decrease the WSTETH KFb Metaparameter - February 19, 2024.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Decrease the WSTETH KFb Metaparameter - February 19, 2024
+summary: Signal your support or opposition to decrease the WSTETH KFb metaparameter.
+discussion_link: https://forum.makerdao.com/t/atlas-rate-system-iteration-1-adjusting-wsteth-asset-spread/23687
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-02-19T16:00:00
+end_date: 2024-02-22T16:00:00
+---
+# Poll: Decrease the WSTETH KFb Metaparameter - February 19, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Scope Facilitator. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, February 19 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
+
+- Decrease the WSTETH [`KFb` metaparameter](https://mips.makerdao.com/mips/details/MIP104#14-3-1-3-stability-fee-sf-) by 6% from 13.5% to **7.5%**.
```

Comment:
Let's remove this to prevent conflicts with the other poll.

```suggestion
- Decrease the WSTETH [`KFb` metaparameter](https://mips.makerdao.com/mips/details/MIP104#14-3-1-3-stability-fee-sf-) to **7.5%**.
```